docs/ANONYMITY_NETWORKS.md

```diff
@@ -1,149 +1,176 @@
-received at address "rveahdfho7wo4b2m.onion:28083" and forwarded to `monerod`
-localhost port 28083, and a default max I2P connections are being received at
-address "cmeua5767mz2q5jsaelk2rxhf67agrwuetaso5dzbenyzwlbkg2q.b32.i2p:5000" and
-forwarded to `monerod` localhost port 30000.
-These addresses will be shared with outgoing peers, over the same network type,
-otherwise the peer will not be notified of the peer address by the proxy.
 
-### Wallet RPC
+## Inbound connections
 
-An anonymity network can be configured to forward incoming connections to a
-`monerod` RPC port - which is independent from the configuration for incoming
-P2P anonymity connections. The anonymity network (Tor/i2p) is
-[configured in the same manner](#configuration), except the localhost port
-must be the RPC port (typically 18081 for mainnet) instead of the p2p port:
+An anonymity network can be configured to forward incoming connections to `monerod`. Add the option
+```
+--anonymous-inbound rveahdfho.onion:18083,127.0.0.1:18083,25
+--anonymous-inbound cmeua5767.b32.i2p:5000,127.0.0.1:30000
+```
+which tells `monerod` that a max of 25 inbound Tor connections are being
+received at address `rveahdfho.onion:18083` and forwarded to `monerod`
+localhost port `18083`.  
+In this case for I2P, the default max. connections are being received at address `cmeua5767.b32.i2p:5000` and forwarded to `monerod` localhost port `30000`. The connecting hidden service address will be seeded to other peers on the AN.
 
+Incoming connections require a hidden service. Regards Tor, add
 ```
+# file /etc/tor/torrc
 HiddenServiceDir /var/lib/tor/data/monero
-HiddenServicePort 18081 127.0.0.1:18081
+# HiddenServicePort 18081 127.0.0.1:18081 # rpc port
+HiddenServicePort 18083 127.0.0.1:18083   # tx port
 ```
+and restart `sudo systemctl restart tor.service`. This will generate your `.onion` address in `/var/lib/tor/data/monero/hostname`. 
+
+I2P must be configured with a standard server tunnel. Configuration differs by I2P implementation.
+
+
+### Allow inbound RPC
 
-Then the wallet will be configured to use a Tor/i2p address:
+The clients `monero-wallet-cli` and `monero-wallet-rpc` can connect to a daemon over AN. The daemon must provide a hidden service on free port that redirects the request to your local rpc port `--rpc-bind-ip`.
 ```
---proxy 127.0.0.1:9050
---daemon-address rveahdfho7wo4b2m.onion
+HiddenServicePort 18081 127.0.0.1:18081 # rpc port
 ```
+Now not only localhost rpcs, but remote rpcs incoming from AN via port 18081 can use your service. Use with caution.
 
-The proxy must match the address type - a Tor proxy will not work properly with
-i2p addresses, etc.
+If you wish to run a public rpc service, further options for `monerod` are availble that restrict available commands, add rpc-ssl or require authentication. You can also require e.g. [Client Authoriztion](https://community.torproject.org/onion-services/advanced/) on the AN side.
```

Comment:
"authorization" is misspelled. 

I would word this section differently - if the user does not specify some client authorization, then the RPC service is basically public (the server/port is listed in the available hidden services list). So the wording should steer users to adding some authentication.